.github/SUPPORT.md

```diff
@@ -1,26 +1,108 @@
-# Support
+# Node.js Support Guide
 
 Node.js contributors have limited availability to address general support
-questions. Please make sure you are using a [currently-supported version of
-Node.js](https://github.com/nodejs/Release#release-schedule).
+questions. Please make sure you are using a
+[currently-supported version of Node.js][1].
 
-When looking for support, please first search for your question in these venues:
+When seeking support, please first search for your question in these venues:
 
-* [Node.js Website](https://nodejs.org/en/), especially the
-  [API docs](https://nodejs.org/api/)
-* [Node.js Help](https://github.com/nodejs/help)
-* [Open or closed issues in the Node.js GitHub organization](https://github.com/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+org%3Anodejs+is%3Aissue)
+- [Node.js Website][2], especially the [API docs][3]
+- [Node.js Help][4]
+- [Open or closed issues in the Node.js GitHub organization][5]
 
 If you didn't find an answer in the resources above, try these unofficial
 resources:
 
-* [Questions tagged 'node.js' on Stack Overflow](https://stackoverflow.com/questions/tagged/node.js)
-* [#node.js channel on libera.chat](https://web.libera.chat?channels=node.js&uio=d4)
-* [Node.js Slack Community](https://node-js.slack.com/)
-  * To register: [nodeslackers.com](https://www.nodeslackers.com/)
+- [Questions tagged 'node.js' on Stack Overflow][6]
+- [#node.js channel on libera.chat][7]
+- [Node.js Slack Community][8] *(Register at [nodeslackers.com][9])*
 
 GitHub issues are for tracking enhancements and bugs, not general support.
 
 The open source license grants you the freedom to use Node.js. It does not
 guarantee commitments of other people's time. Please be respectful and manage
 your expectations.
+
+## Troubleshooting Steps
+
+When encountering issues with Node.js, follow these troubleshooting steps:
+
+1. **Check Node.js Version**: Ensure you are using a currently-supported
+version of Node.js. You can check the supported versions [here][1].
```

Comment:
Having "here" as the text of the link is bad practice (mandatory link to https://userinyerface.com/ to illustrate this)

Reply:
*(resolved locally)*